ros2_control_test_assets/include/ros2_control_test_assets/components_urdfs.hpp

```diff
@@ -0,0 +1,568 @@
+// Copyright (c) 2021 Stogl Robotics Consulting
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROS2_CONTROL_TEST_ASSETS__COMPONENTS_URDFS_HPP_
+#define ROS2_CONTROL_TEST_ASSETS__COMPONENTS_URDFS_HPP_
+
+#include <string>
+
+namespace ros2_control_test_assets
+{
+
+const auto urdf_xml_head_ =
```

Comment:
what's the difference to the `urdf_xml_head` in the other urdf header file in the same repo? I think we could recycle that one, also makes the diff smaller.